### PR TITLE
Reworked login and register scripts and combined them into one sending formula scripts

### PR DESCRIPTION
**Pull request by tomazaw**
Also fixed several bugs and user settings are sending properly.

### CODE REVIEW

File: public/scripts/form-send-script.js [hunk 1]
```diff
@@ -0,0 +1,75 @@
+function formListener(formName, path) {
+window.addEventListener('load', () => {
+   function sendForm(path) {
+		const request = new XMLHttpRequest();
```
Comment (MasterCoookie) [inline public/scripts/form-send-script.js]: indenty

File: public/scripts/form-send-script.js [hunk 2]
```diff
@@ -0,0 +1,75 @@
...
+            }
+            else if(path === '/auth/changePassword' || path === '/auth/changeEmail'){
+                if (response.status === 200) {
+                    document.getElementById('message').innerHTML = response.msg;
+                }
+                else {
+                    document.getElementById('message').innerHTML = 'Something went wrong.';
+                }
+            }
+		});
```
Comment (MasterCoookie) [inline public/scripts/form-send-script.js]: tu

File: public/scripts/form-send-script.js [hunk 3]
```diff
@@ -0,0 +1,75 @@
...
+            }
+            else {
+                document.getElementById('message').innerHTML = "Both passwords are not matching.";
+            }
+        }
+        else {
+            request.open('POST', path, true);
+            request.send(formData);
+        }
+	}
```
Comment (MasterCoookie) [inline public/scripts/form-send-script.js]: tu

File: public/scripts/form-send-script.js [hunk 4]
```diff
@@ -0,0 +1,75 @@
...
+                request.open('POST', path, true);
+                request.send(formData);
+            }
+            else {
+                document.getElementById('message').innerHTML = "Both passwords are not matching.";
+            }
+        }
+        else if(path === '/auth/register'){
+        var firstPassword = formData.get('passwordInput');
+		var repeatedPassword = formData.get('repeatPasswordInput');
```
Comment (MasterCoookie) [inline public/scripts/form-send-script.js]: tu

File: public/scripts/form-send-script.js [hunk 5]
```diff
@@ -0,0 +1,75 @@
...
+            if(firstPassword === repeatedPassword) {
+            request.open('PUT', '/auth/register', true);
+            request.send(formData);
```
Comment (MasterCoookie) [inline public/scripts/form-send-script.js]: tu

**Review by MasterCoookie — CHANGES_REQUESTED** [review]
popraw indenty pls

**Review by MasterCoookie — APPROVED** [review]

**Review by Direckton — APPROVED** [review]

**Pull request merged**